<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Межфондовый указатель к документам духовного ведомства периода до октября 1917 года</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_aboutqt"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Справочники</string>
    </property>
    <addaction name="action_doctype"/>
    <addaction name="action_docflag"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Сервис</string>
    </property>
    <addaction name="action_dbsettings"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/exit-16.png</normaloff>:/icons/exit-16.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_aboutqt">
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/search-file-16.png</normaloff>:/icons/search-file-16.png</iconset>
   </property>
   <property name="text">
    <string>Поиск по указателю</string>
   </property>
  </action>
  <action name="action_doctype">
   <property name="text">
    <string>Виды документов</string>
   </property>
  </action>
  <action name="action_docflag">
   <property name="text">
    <string>Доп. сведения (флаги)</string>
   </property>
  </action>
  <action name="action_dbsettings">
   <property name="text">
    <string>Настройка подключения</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
